deploy.sh

```diff
@@ -2,10 +2,11 @@
 set -e # Fail script if a command fails
 
 # Start the ssh agent (required to clone/pull)
-eval $(ssh-agent -s)
+eval "$(ssh-agent -s)"
 ssh-add <(cat ~/.ssh/deploy_pull)
 
 # Load environment variables passed in from CI container
+# shellcheck source=/dev/null
```

Comment:
what does that mean?

Reply:
https://github.com/koalaman/shellcheck/wiki/SC1090 it can't know what variables are set by it. I don't care so used /dev/null